engine/src/test/java/org/destinationsol/testsupport/AssetsHelperInitializer.java

```diff
@@ -0,0 +1,57 @@
+package org.destinationsol.testsupport;
+
+import com.badlogic.gdx.Application;
+import com.badlogic.gdx.ApplicationAdapter;
+import com.badlogic.gdx.backends.headless.HeadlessApplication;
+import org.destinationsol.assets.AssetHelper;
+import org.destinationsol.assets.Assets;
+import org.destinationsol.modules.ModuleManager;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.parallel.ResourceLock;
+import org.terasology.gestalt.entitysystem.component.management.ComponentManager;
+
+/**
+ * Use it for tests, which used {@link org.destinationsol.assets.Assets}
+ */
+@ResourceLock("AssetsHelper")
+public interface AssetsHelperInitializer {
+    ThreadLocal<TestGameStateObject> state = new ThreadLocal<>();
+
```

Comment:
Some Javadoc here would be helpful for figuring out what exactly this does. From what I can tell, this sets up certain assets for a test, but I'm not clear exactly what. Something like: "Initializes x, y, and z, so that cases such as a, b, and c have access to those resources without building a full game object."

Reply:
It is interfaces. interface can store only `public` `static` fields.
Then we should to use `ThreadLocal` + Before/After to store field only for  this Test.
It is not specific usage nor my created idea just now.
It is like standard for interfaces with defaults in junit5.

Give me time after this PR. then I will create Junit5's Extension for tests. ok?